test/test-ncu.js

```diff
@@ -95,14 +94,21 @@ describe('npm-check-updates', function () {
         });
 
         it('should fall back to package.json search when receiving empty content on stdin', function (done) {
-            var childProcess = child.exec('node bin/ncu', function (error, stdout) {
-                if (error) {
-                    done(error);
-                }
+            spawn('node', ['bin/ncu']).then(function (stdout) {
                 stdout.toString().trim().should.match(/^Using .+package.json/);
                 done();
             });
-            childProcess.stdin.end();
+        });
+
+        it('should handle no package.json to analyze when receiving empty content on stdin', function (done) {
```

Comment:
Sorry I didn't catch this before, but is there any reason to use `pwd` here instead of just using `__dirname`?